haproxy-redundant-floatingip-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,506 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "azureuser",
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "sshKeyData": {
+      "type": "string",
+      "metadata": {
+        "description": "SSH rsa public key file as a string."
+      }
+    },
+    "dnsNameforLBIP": {
+      "type": "string",
```

Comment:
Call this a dnsPrefix instead. See #966. Also remove the default value since this needs to be unique